iwyu.cc

```diff
@@ -1728,6 +1729,11 @@ class IwyuBaseAstVisitor : public BaseAstVisitor<Derived> {
         ReportDeclUse(CurrentLoc(), redecl, nullptr, UF_DefinitionUse);
       }
     }
+
+    if (const CleanupAttr *attr = decl->getAttr<CleanupAttr>()) {
```

Comment:
```suggestion
    if (const auto* attr = decl->getAttr<CleanupAttr>()) {
```
Besides, it's worth to run clang-format on the changes.